python/tvm_ffi/module.py

```diff
@@ -322,12 +323,20 @@ def load_module(path: str) -> Module:
     --------
     .. code-block:: python
 
-      mod = tvm_ffi.load_module("path/to/module.so")
-      mod.func_name(*args)
+        # Works with string paths
+        mod = tvm_ffi.load_module("path/to/module.so")
+        mod.func_name(*args)
+
+        # Also works with pathlib.Path objects
+        from pathlib import Path
+        mod = tvm_ffi.load_module(Path("path/to/module.so"))
+        mod.func_name(*args)
 
     See Also
     --------
     :py:class:`tvm_ffi.Module`
 
     """
-    return _ffi_api.ModuleLoadFromFile(path)
+    # Convert PathLike objects to string
```

Comment:
It should be sufficient to just use `path = fspath(path)` as in python's stdlib:

https://github.com/python/cpython/blob/bd2c7e8c8b10f4d31eab971781de13844bcd07fe/Lib/posixpath.py#L62-L64